### PR TITLE
DOCSP-34512 update steps for Atlas Search Index Configuration File

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-34512)
- [Stage](https://preview-mongodbkanchanamongodb.gatsbyjs.io/atlas-cli/DOCSP-34512/reference/json/search-index-config-file/)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=657c8af97a5cb5fef58307e7)

### CODE REVIEW

File: source/includes/create-atlas-search-expanded-index-config-file.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      "fields": {
+        "name": {
+            "type": "string"
+        },
+        "property_type": {
+            "type": "string"
+        }
+      }
+    }
+  }
```
Comment (corryroot) [inline source/includes/create-atlas-search-expanded-index-config-file.json:16]: Spacing looks off:
```suggestion
{
  "collectionName": "listingsAndReviews",
  "database": "sample_airbnb",
  "name": "myIndex",
  "mappings": {
    "dynamic": false,
    "fields": {
      "name": {
        "type": "string"
      },
      "property_type": {
        "type": "string"
      }
    }
  }
}
```

File: source/reference/json/search-index-config-file.txt [hunk 2]
```diff
@@ -11,9 +11,12 @@ Atlas Search Index Configuration File
...
 
 You can use an |fts| index configuration file to specify the required 
-settings for :ref:`creating a search index
-<atlas-clusters-search-indexes-create>` using the {+atlas-cli+}. The 
-{+atlas-cli+} accepts ``.json`` search index configuration files.
+settings for creating a search index using the {+atlas-cli+}. The 
+{+atlas-cli+} accepts ``.json`` search index configuration files. You
+can use the  ``.json`` search index configuration file to create the
+index for :ref:`cloud <atlas-clusters-search-indexes-create>` and
+:ref:`local <atlas-deployments-search-indexes-create>` Atlas
```
Comment (corryroot) [inline source/reference/json/search-index-config-file.txt:18]:
```suggestion
:ref:`local <atlas-deployments-search-indexes-create>` |service|
```

**Review by corryroot — CHANGES_REQUESTED** [review]
Thanks @kanchana-mongodb! The spacing looks off in the new example.

**Review by corryroot — APPROVED** [review]
LGTM!

**Review by tibulca — APPROVED** [review]
lgtm

**Pull request merged**